### ISSUE TITLE
🐛 Bug Report: SSL Certificate is not being renewed

### ISSUE THREAD

**Issue by OKatrych** [appwrite/appwrite#3068]
### 👟 Reproduction steps

Have a domain with an issued certificate from Appwrite Let's Encrypt. Wait until the certificate is expired.

### 👍 Expected behavior

The certificate should be automatically renewed before expiration.

### 👎 Actual Behavior

The certificate is expired and I see no instruction on how to renew it [here](https://appwrite.io/docs/certificates).

`docker-compose logs appwrite-worker-certificates` returns this:
```
root@ubuntu-appwrite:~/appwrite# docker-compose logs appwrite-worker-certificates
Attaching to appwrite-worker-certificates
appwrite-worker-certificates    | Appwrite certificates worker v1 has started
appwrite-worker-certificates    | [notice] Starting worker c8e624c3d236:7:v1-certificates
```

### 🎲 Appwrite version

Version 0.13.x

### 💻 Operating system

MacOS

### 🧱 Your Environment

I use DigitalOcean droplet.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by OKatrych** [comment]
Is there any way to run Let'sEncrypt manually?

**Comment by OKatrych** [comment]
Just found that it can be done with the `docker-compose exec appwrite ssl` command. Can it be added into [docs](https://appwrite.io/docs/certificates) as a workaround?

**Comment by OKatrych** [comment]
Also, after running it, from the logs I see:

**Comment by OKatrych** [comment]
After second run:

**Comment by OKatrych** [comment]
And I managed to renew it by making a full reboot of the server. I think this would be great to have this documented as a workaround.
So to renew the certificate I had:

**Comment by TorstenDittmann** [comment]
hey :)

we are actively working right now on improving the certificates with Appwrite in general.

#2997 #3074 #3034

**Comment by eldadfux** [comment]
This issue was fixed and released in Appwrite 0.14
